source/deployment-guide/mobile/configure-microsoft-intune-mam.rst

```diff
@@ -0,0 +1,398 @@
+
+The following rules apply:
+
+* ``IdAttribute`` must equal Azure AD ``objectId``.
+* MSAL access tokens must include the ``oid`` claim.
+* All authentication systems must resolve to the same value.
+
+If any authentication path resolves a different identifier, enrollment will fail.
+
+Supported Identity Attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Only the identity attributes listed below are supported for Intune MAM.
+
++-------------------+------------------+------------------------------+
+| Attribute         | Supported        | Result                       |
++===================+==================+==============================+
+| objectId          | Required         | Works                        |
++-------------------+------------------+------------------------------+
+| email             | Not supported    | Enrollment fails             |
++-------------------+------------------+------------------------------+
+| preferred_username| Not supported    | Identity mismatch            |
++-------------------+------------------+------------------------------+
+| objectGUID        | Not supported    | Breaks mobile authentication |
++-------------------+------------------+------------------------------+
+| Custom attributes | Not supported    | Unsupported by Intune        |
++-------------------+------------------+------------------------------+
+
+Attribute Synchronization & Access Enforcement
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When Intune MAM is enabled, some users may authenticate exclusively through the Mattermost mobile app. If your deployment uses SAML or OpenID Connect, note the following behavior:
+
+* User attributes synchronize only at login.
+* Changes made in the identity provider do not apply until the next login.
+* Mobile-only users may not trigger attribute synchronization.
+
+As a result, attribute-based access control (ABAC) may not apply immediately.
+
+If proactive enforcement of attribute-based access changes is required, we recommend LDAP (including Entra ID Domain Services). This behavior affects **access enforcement**, not Intune MAM enrollment.
+
+Runtime Enforcement Behavior
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost Mobile enforces Intune MAM requirements during active sessions, not only at login.
+
+If Intune MAM becomes newly required due to policy, licensing, or configuration changes:
+
+* Enrollment is triggered immediately.
+* Access to sensitive content is restricted until enrollment succeeds.
+* Users cannot bypass enforcement.
+
+Plan rollouts assuming enforcement can occur instantly.
+
+Once your identity model and enforcement behavior are understood and aligned, ensure the following prerequisites are in place before beginning configuration.
+
+Detailed Prerequisites
+----------------------
+
+Microsoft Requirements
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* Microsoft Entra tenant
+* Permissions to register and manage enterprise applications
```

Comment:
not sure how enterprise applications and App registrations differ, but they will need access to both